crates/core/tedge_actors/src/errors.rs

```diff
@@ -44,11 +47,14 @@ where
 }
 
 /// Error raised while connecting actor instances
-#[derive(Error, Debug, Clone, PartialEq, Eq)]
+#[derive(Error, Debug)]
```

Comment:
There is no reasons to remove these trait implementations.

Reply:
fixed